3 - PB/Documentazione esterna/Verbali/24-02-22/24-02-22.typ

```diff
@@ -0,0 +1,85 @@
+
+== Feedback su modalità e strumenti per il testing
+
+Il gruppo ha esposto al Proponente le proprie considerazioni riguardanti il testing dell'applicazione. Inizialmente le tipologie di testing considerate erano:
+
+- unit testing;
+- integration testing;
+- end-to-end testing.
+
+=== unit testing
+
+Il Proponente, seppur indicando che in azienda viene utilizzato JUnit per il processo di unit testing, non ha posto vincoli riguardo al software da adottare ed ha apprezzato la proposta del gruppo di utilizzare Jest.
+Il Proponente ha poi consigliato di rendere questa modalità di testing il focus principale del gruppo, poiché ritenuta di importanza maggiore rispetto alle altre.
+
+=== integration testing
+
+Il Proponente ha manifestato la visione che lo sviluppo di integration testing non sia considerato un requisito obbligatorio, ribadendo che la logica di business non costituisce il fulcro principale del capitolato proposto.
+
+Il gruppo, a seguito di questa indicazione, si impegna a prendere una decisione finale in merito durante lo svolgimento dei lavori. Si prevede un'approfondita valutazione della necessità e dei potenziali benefici legati allo sviluppo di integration testing. La decisione definitiva sarà presa in considerazione considerando l'orientamento del Proponente e valutando attentamente come questa scelta possa influire sulle fasi successive del progetto.
+
+
+=== end-to-end testing
+
+Il Proponente ha espresso interesse nei confronti della proposta, ritenendo tuttavia che lo sviluppo di questo tipo di testing non sia attualmente una priorità. Ha sottolineato la percezione di un'elevata richiesta in termini di tempo e risorse, considerandolo un'impostazione che potrebbe risultare troppo esosa nell'attuale contesto di sviluppo.
+
+In seguito a questa valutazione da parte del Proponente, il gruppo ha preso atto della posizione e si riserva il diritto di prendere una decisione finale in merito durante lo svolgimento dei lavori. In particolare, verrà dedicato uno spazio specifico per valutare l'implementazione del software Selenium suggerito dal Proponente, con l'obiettivo di esaminare più approfonditamente i potenziali benefici e i requisiti di tempo e risorse associati. La decisione finale sarà presa considerando attentamente tutti questi fattori.
+
+== feedback sul deploy dell'applicazione
+
+Il Proponente ha ritenuto interessante la proposta del gruppo di implementare il processo di deploy dell'applicazione utilizzando AzureVM (oltre a Docker).
+Il feedback generale ha evidenziato l'approvazione dei meccanismi di Continuous Integration e Continuous Deployment (CI/CD) implementati durante questa fase.
+Nel corso del deploy su Azure, il gruppo ha osservato la stabilità e le prestazioni della piattaforma e l'efficacia del sistema di CI/CD nel ridurre i tempi di rilascio e migliorare la gestione delle nuove funzionalità.
```

Comment:
Non aggiunge informazione questa frase :/
Durante il meeting vero e proprio non abbiamo mostrato alcun deploy ne abbiamo solo discusso in merito all'intenzione di farlo e di come lo abbiamo fatto per rendere il prodotto sempre accessibile.